src/components/_legal/FilesComponent/FilesComponent.types.ts

```diff
@@ -0,0 +1,9 @@
+export type Props = {
+  data: Array<{
+    asset: {
+      url: string;
+      originalFilename: string;
+      size: number;
+    }
+  }>;
```

Comment:
Use one convention. Above you've use `{}[]`, and now you're using `Array<{}>`.
Let's make an agreement to always use `{}[]` in TypeScript to mark arrays.